src/server/cluster/cluster_family_test.cc

```diff
@@ -625,7 +625,12 @@ TEST_F(ClusterFamilyTest, ClusterModePubSubNotAllowed) {
               ErrArg("PSUBSCRIBE is not supported in cluster mode yet"));
   EXPECT_THAT(Run({"PUNSUBSCRIBE", "ch?"}),
               ErrArg("PUNSUBSCRIBE is not supported in cluster mode yet"));
-  EXPECT_THAT(Run({"PUBSUB", "CHANNELS"}), ErrArg("PUBSUB is not supported in cluster mode yet"));
+  EXPECT_THAT(Run({"PUBSUB", "CHANNELS"}),
```

Comment:
Add the opposite checks for non cluster mode. That is `PUBSUB SHARDCHANNELS` gets rejected in `non cluster mode`

Reply:
Ignore me, I missed that you included those. :100:

Reply:
Using SHARDCHANNELS and SHARDNUMSUB in non cluster mode will work, i'll add this check and update tests